pkg/storage/storage.go

```diff
@@ -16,8 +16,9 @@ const (
 )
 
 type PaginationOptions struct {
-	PageSize int
-	From     string
+	PageSize      int
+	From          string
+	OrderRequired bool
```

Comment:
Didn't we discuss removing OrderRequired and use the presence of Pagination options instead?

If there is any pagination then it necessitates ordering

Reply:
Having no pagination options does not necessary mean that we should not have order (i.e., page 0 and default page size).

Reply:
But in that case there still are paginationOptions (with the options being 0 and default size)

paginationOptions itself is not nil